src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/activity/ActivityPageFactory.java

```diff
@@ -1,29 +1,24 @@
 package com.wikia.webdriver.pageobjectsfactory.componentobject.activity;
 
-import com.wikia.webdriver.pageobjectsfactory.componentobject.activity.creators.ActivityPageCreator;
-import org.openqa.selenium.WebDriver;
+
+import lombok.Getter;
 import org.openqa.selenium.WebElement;
-import javax.swing.text.html.HTML;
-import java.util.List;
 
+import java.util.ArrayList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)